deployment/terraform/storage.tf

```diff
@@ -1,18 +1,12 @@
 #
 # S3 resources
 #
-# resource "aws_s3_bucket" "logs" {
-#   bucket = "rastervision-${lower(var.environment)}-logs-${var.aws_region}"
-#   tags {
-#     Project     = "${var.project}"
-#     Environment = "${var.environment}"
-#   }
-# }
-# resource "aws_s3_bucket" "catalogs" {
-#   bucket = "rastervision-${lower(var.environment)}-catalogs-${var.aws_region}"
-#   tags {
-#     Project     = "${var.project}"
-#     Environment = "${var.environment}"
-#   }
-# }
 
+resource "aws_s3_bucket" "catalogs" {
```

Comment:
Is something supposed to be in this bucket?

Reply:
@lewfish, does it make sense to copy just the contents of this subdirectory over to a bucket within the GeoTrellis AWS account? The goal is to try and keep things this service depends on within the same account, despite the fact that the GeoTrellis AWS account can read/write to the R&D account.

Reply:
That makes sense to me, but I had very little to do with this repo, so I think we should see what @lossyrob thinks.

Reply:
Yep, there's no issue with copying this data over to the GeoTrellis account.